### ISSUE TITLE
[$15][MSFT-14] No “alt” text present on the third party OAuth providers

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2738]
**Steps to Reproduce**
1. Go to https://accounts.topcoder.com/member
2. Go to Browser settings -> Privacy -> Images and disable images for the above link.
3. Reload the page.
4. You will see there is no “alt” text available for the third party OAuth provider images.
5. The VoiceOver in macOS was not able to recognise the provider.

**Expected Result**
The images should have the alt text. This way, the screen readers will be able to recognise the provider.

**Actual Result**
There is no alt text in the OAuth provider images. VoiceOver in macOS failed to recognise the provider.

**Device**: Desktop/Labtop
**Operating System**: macOS
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.1.1 Non-Text Content
**Screen Reader Used**: Voiceover
**Accessibility Audit Tool used**: 

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=1f11c1b0%2Dabf6%2D41bc%2Dbbc3%2D8435a17fcc67&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fanmolwassan%2FBug%201

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096320 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096320 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096320 has been updated - it has been assigned to paccode.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by dzanchett** [comment]
@veshu PR appirio-tech/accounts-app#251

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096320 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by mishacucicea** [comment]
Adjusting prize as it impacts the Register page as well. @dzanchett see details in the code review.

**Comment by lakshmiathreya** [comment]
Verified with MacOS/Chrome (voiceover), Win10 Chrome/FF (Jaws) - tab focus and screen reader work correctly for all OAth provider links.

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2738 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2738 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097460 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097460 has been updated - it has been assigned to paccode.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by lakshmiathreya** [comment]
Verified on Mac/Chrome with VoiceOver. Working fine. @drasticdpk pls update and close after testing on Windows.

**Comment by drasticdpk** [comment]
@mishacucicea  @lakshmiathreya , Screen reader is working as expected on Windows-JAWS/NVDA too. But ALT text is not showing while disabling the image , here is the reference image. 
![image](https://user-images.githubusercontent.com/25297859/61996252-04a37800-b0b2-11e9-9581-f657d9250be9.png)

**Comment by mishacucicea** [comment]
@dzanchett You need to add title on img as well so that it shows correctly (https://stackoverflow.com/questions/5386570/how-to-display-alt-text-for-an-image-in-chrome)

**Comment by dzanchett** [comment]
@mishacucicea Title added. I also had to edit the `app/styles/tc/account.scss` file to solve this.

**Comment by nkumar-topcoder** [comment]
@drasticdpk @mishacucicea  @lakshmiathreya This has been re-fixed post "tcx_feedback"

**Comment by drasticdpk** [comment]
Verified.

**Comment by drasticdpk** [comment]
verified on MAC/Chrome

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097460<br/><br/>```This is an automated message for crazyk via Topcoder X```
